src/view/DocumentTargetPicker.vue

```diff
@@ -5,7 +5,7 @@
 
 <template>
 	<div v-if="filePath === null" class="office-target-picker">
-		<div ref="picker" class="reference-file-picker" />
+		<div class="reference-file-picker" />
```

Comment:
We could actually use the vue component directly https://github.com/nextcloud-libraries/nextcloud-dialogs?tab=readme-ov-file#use-the-vue-component-directly

That way we would not have nested modals

Reply:
Indeed, it has been improved, however the FilePickerVue component itself is actually wrapped in an NcDialog
https://github.com/nextcloud-libraries/nextcloud-dialogs/blob/016d046e79b949838bb78d01dca9bcb1e1db286a/lib/components/FilePicker/FilePicker.vue#L6

Reply:
All right, then fine either way 😄